sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/OperationalLevelTimeoutExample.java

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob;
+
+import com.azure.core.http.HttpPipelineCallContext;
+import com.azure.core.http.HttpPipelineNextPolicy;
+import com.azure.core.http.HttpPipelinePosition;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import com.azure.storage.blob.options.BlobContainerCreateOptions;
+import com.azure.storage.common.StorageSharedKeyCredential;
+import reactor.core.publisher.Mono;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Locale;
+
+/**
+ * This example shows how to start using the Azure Storage Blob SDK for Java.
```

Comment:
This should be more descriptive and include that these timeouts are operation level timeouts.